### PR TITLE
feat(exchange): emphase on the filter button

### LINKED ISSUE #1695: feat(exchange): emphase on the filter button
- [x] should update the filter button design following the design

### PR DESCRIPTION
**Pull request by JNSPK**
<p align =center>${\color{lightgreen}FIX}$ </p></p><p align=center></p><p align =right>fix #1695 </p>

### CODE REVIEW

File: apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx [hunk 1]
```diff
@@ -24,17 +24,17 @@ export default function FilterForm({
   return (
     <div className={base({ className })} {...props}>
       <Link
-        className={button({
+        className={`${button({
           active: filter_query === filter_params_schema.enum.IN_PROGRESS,
-        })}
+        })} bg-secondary hover:bg-secondary`}
```
Comment (douglasduteil) [inline apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx:29]: Prefer 
```tsx
className={button({
  active: filter_query === filter_params_schema.enum.SUCCESS,
  intent: "secondary"
})}
```

File: apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx [hunk 2]
```diff
@@ -24,17 +24,17 @@ export default function FilterForm({
...
         href={{ query: in_progress_params.toString() }}
       >
         En cours
       </Link>
       <Link
-        className={button({
+        className={`${button({
```
Comment (douglasduteil) [inline apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx:35]: Prefer 
```tsx
className={button({
  active: filter_query === filter_params_schema.enum.SUCCESS,
  intent: "primary"
})}
```

File: apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx [hunk 3]
```diff
@@ -67,15 +67,22 @@ const search_form_classes = tv({
   base: `grid grid-cols-2 gap-4`,
   slots: {
     button: ui_button({
-      className: "whitespace-nowrap",
-      intent: "whity",
+      className: "w-1/2 whitespace-nowrap",
```
Comment (douglasduteil) [inline apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx:70]:
```suggestion
      className: "w-1/2 whitespace-nowrap",
```

Can you find another way ? The button is an inline-block right ?

File: apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx [hunk 4]
```diff
@@ -67,15 +67,22 @@ const search_form_classes = tv({
...
-        button: "font-normal opacity-75",
+        button: "font-normal opacity-50  hover:opacity-100",
+      },
+      true: {
+        button: "font-bold opacity-100",
+      },
+    },
+    hover: {
+      true: {
+        button: "hover:bg-secondary",
```
Comment (douglasduteil) [inline apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx:85]:
```suggestion
```

Seems unused

**Review by douglasduteil — CHANGES_REQUESTED** [review]
<p align="center">
    
</p>

File: apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx (hunk 1 above)
Comment (JNSPK) [reply apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx]: Done. Does it suits you ?

File: apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx (hunk 2 above)
Comment (JNSPK) [reply apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx]: Done. Does it suits you ?

File: apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx (hunk 3 above)
Comment (JNSPK) [reply apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx]: Flex-1 Done. Does it suits you ?
#FLEXFORLIFE

File: apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx (hunk 4 above)
Comment (JNSPK) [reply apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx]: True that ! Done.

**Pull request merged**